examples/meteor/client/styles/base.css

```diff
@@ -285,9 +284,9 @@ label[for='toggle-all'] {
 	display: block;
 }
 
-#todo-list li .edit {
-	display: none;
-}
+/*#todo-list li .edit {*/
+	/*display: none;*/
+/*}*/
```

Comment:
Using CSS to show/hide the edit field is "old school" in terms of Meteor. You can see in the code that I use an `{{#if}}` statement to show/hide the edit field, which is a better way of handling this.